Gopkg.toml

```diff
@@ -81,3 +82,7 @@ required = [
 [[constraint]]
   branch = "master"
   name = "github.com/wadey/gocovmerge"
+
+[[constraint]]
```

Comment:
I believe we are not using protobuf as direct dependency, we should use `[[override]]` here

Reply:
Thanks @dipak-pawar, that's a good point. I fixed that...

Reply:
@pamcik, thanks, it'll be good if you can update `Gopkg.lock` to ensure a reproducible build?